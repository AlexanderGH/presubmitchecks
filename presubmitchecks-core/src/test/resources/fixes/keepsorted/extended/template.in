Template param is expanded:
    // keep-sorted-test start template=gradle.kts
    implementation(b.c) {
        because("c")
    }
    implementation(a.a) {
        because("a")
    }
    implementation(a.b) {
        because("b")
    }
    // keep-sorted-test end
