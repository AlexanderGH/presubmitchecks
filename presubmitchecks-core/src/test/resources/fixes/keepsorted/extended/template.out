Template param is expanded:
    // keep-sorted-test start template=gradle.kts
    implementation(a.a) {
        because("a")
    }
    implementation(a.b) {
        because("b")
    }
    implementation(b.c) {
        because("c")
    }
    // keep-sorted-test end